.circleci/config.yml

```diff
@@ -95,6 +117,11 @@ workflows:
   validate_spec:
     jobs:
       - build_and_test:
+          matrix:
+            parameters:
+              os: [docker]
+              node-version: ["^14.20.0", "^16.17.0", "^18.9.0"]
```

Comment:
You don't need to create all these matrices for the integration test, only one node_version is sufficient.

Reply:
I updated see line 128.